src/Components/Display.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import '../Styles/Display.css';
+
+const Display = (props) => {
+  const { result } = props;
+  return (
+    <div>
+      <h2>{result}</h2>
+    </div>
+  );
+};
+
+Display.propTypes = {
+  result: propTypes.string,
+};
+Display.defaultProps = {
+  result: 0,
```

Comment:
-[ ] its  should be a String with default value = 0 [should be String screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png) . Kindly put your `0` in the string